saleor/graphql/checkout/tests/test_checkout.py

```diff
@@ -2639,3 +2639,40 @@ def test_get_checkout_with_vatlayer_set(
     # then
     content = get_graphql_content(response)
     assert content["data"]["checkout"]["token"] == str(checkout.token)
+
+
+def test_checkout_customer_attach_user_to_checkout_with_user(
```

Comment:
It's a good practice to keep related tests close to each other. We have `test_checkout_customer_attach` in this file; we could move this one there.